### DISCUSSION TITLE
number of azimuth looks for ALOS-2 ScanSAR processing

### DISCUSSION THREAD

**Discussion by fdelgadodelapuente** [isce-framework/isce2#788] [General]
Dear ISCE developers, 

I have a question about the ALOS-2 ScanSAR processing. Why is the numberAzimuthLooks1 set to the default 14? Is it just an empirical estimate to retrieve approximately square pixels after taking looks in range or does it come from somewhere else? In Liana and Fielding (2016) I found the following "the ScanSAR azimuth bandwidth is only about 1/14 PRF". 

Thanks,
Francisco

**Comment by EJFielding** [comment]
Hi Francisco, yes, the choice of 14 azimuth looks in `numberAzimuthLooks1` for the ALOS-2 ScanSAR processing is an empirical estimate of the actual azimuth resolution of the data. @CunrenLiang could say more.
++Eric

**Comment by CunrenLiang** [comment]
The full azimuth aperture is shared by five subswaths. In the share of each subswath, the aperture is further shared by about four looks or bursts, because a target  can be imaged by four bursts in each swath. This is different from Sentinel-1 TOPS, where a target is only imaged by one burst. Therefore, compared with the azimuth burst bandwidth, the full-aperture data is actually oversampled by a factor of about 4*5*f, where f is the oversampling factor as if the data are acquired in stripmap mode. So we can even take 20 looks to remove the oversampling. We actually used 14 because when we mosaic the subswath interferograms, we have to resample the subswath interferograms. I find the resampling results are better if first using a larger oversampling factor and then taking further looks.
